src/locales/de.json

```diff
@@ -2,14 +2,33 @@
 "3d_ie11_alert": "Sie verwenden InternetExplorer <= 11: Die 3D Darstellung wird nicht performant sein. Bitte nutzen Sie die neueste Version von Microsoft EDGE, Firefox oder Chrome um beste Ergebnisse zu erhalten",
 "3d_overlay_warning": "Wichtiger Hinweis: 2D-Daten in 3D.<br/>Sie betrachten jetzt Daten, die für die Darstellung in 3D nicht geeignet sind. Dies gilt insbesondere für Karten und Luftbilder. Diese werden in steilem Gelände bei der Nutzung in 3D verzerrt dargestellt\n",
 "3d_render_error": "Systemvorraussetzung für 3D Ansicht nicht erfüllt. Damit die 3D-Darstellung funktioniert, muss Ihr Gerät folgende Anforderungen erfüllen.Nutzen Sie die neueste Version von Microsoft EDGE, Firefox oder Chrome. Verwendung einer aktuellen Grafikkarte mit aktuellen Treibern. .",
+"Add layer": "Layer hinzufügen",
```

Comment:
Yes it caused by ngeo.

Don't forget de.json is auto generated.
In empty.json file (in the excel file too) we keep the same naming conventions:
```
"add_layer": "Add layer",
```
The text "Add layer" represents the corresponding message id used by ngeo.